src/setupTests.js

```diff
@@ -1,6 +1,20 @@
-global.console.warn = message => {
+global.console.warn = (message) => {
   throw message;
 };
-global.console.error = message => {
+global.console.error = (message) => {
   throw message;
 };
+
+class ResizeObserver {
+    // eslint-disable-next-line class-methods-use-this
+    observe() {
+        // do nothing
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    unobserve() {
+        // do nothing
+    }
+}
+
+window.ResizeObserver = ResizeObserver;
```

Comment:
I am confused about the purpose of this section, why does it need to be included?